Source/santad/EventProviders/DiskArbitrationTestUtil.mm

```diff
@@ -36,15 +36,15 @@ - (instancetype _Nonnull)init {
 - (void)reset {
   [self.insertedDevices removeAllObjects];
   [self.diskAppearedCallbacks removeAllObjects];
+  self.sessionQueue = nil;
   self.wasRemounted = NO;
 }
 
 - (void)insert:(MockDADisk *)ref bsdName:(NSString *)bsdName {
   self.insertedDevices[bsdName] = ref;
 
-  for (NSValue *value in self.diskAppearedCallbacks) {
-    DADiskAppearedCallback callback = (DADiskAppearedCallback)[value pointerValue];
-    callback((__bridge DADiskRef)ref, (__bridge void *)self);
+  for (MockDADiskAppearedCallback callback in self.diskAppearedCallbacks) {
+    callback((__bridge DADiskRef)ref);
```

Comment:
Call this on self.sessionQueue:

```objc
dispatch_sync(self.sessionQueue, ^{ callback((__bridge DADiskRef)ref); });
```

You'll probably need to initialize self.sessionQueue to one of the default queues, to match DA behavior.